### ISSUE TITLE
carve in IFC++

### ISSUE THREAD

**Issue by berndhahnebach** [ifcquery/ifcplusplus#70]
Hi Fabian,

may I ask where do you get the Carve from you gone use in Ifc++  On github I found these two:

https://github.com/qnzhou/carve/commits/master
https://github.com/folded/carve/commits/master

cheers bernd

**Comment by ifcquery** [comment]
Hi Bernd,

ich hab es von
https://github.com/folded/carve/commits/master
Scheint nicht wirklich aktiv gewartet geschweige denn entwickelt zu werden.

Es gibt noch einen Fork, der aber auch nicht aktiver ist:
https://github.com/arnholm/abmesh-carve

Cheers,
Fabian


---------------------------------------------
Fabian Gerold
Mail: fabian.gerold@gmail.com
Phone: (+49)176 55119954

2017-12-19 21:07 GMT+01:00 Bernd Hahnebach <notifications@github.com>:

**Comment by berndhahnebach** [comment]
Hi Fabian,

thanks for the informations.

I have spent some time lately to be able to compile latest IFC++ in Linux. I compiled osg against Qt5 and Ifc++ against Qt5. I still stuggle with Carve (https://github.com/ifcquery/ifcplusplus/issues/63) . Thus I was interested in this information in the ragard of carv repo. For now I reverted Carve to https://github.com/ifcquery/ifcplusplus/commit/463f578dcf2ec1c930ef2351e5baefd72184270e .  I'm quite happy I can again use latest Ifc++  I may spent some time to get it working with latest carve too. Mor information later on.

cheers bernd
